openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -1191,29 +1193,115 @@ Cache-Control: no-store
 }
 ```
 
+# Notification Endpoint {#notification_endpoint}
+
+This endpoint is used by the Wallet to notify the Credential Issuer of certain events for issued credentials. These events enable the Credential Issuer to take subsequent actions after issuance. The Credential Issuer needs to return the `notification_id` in the Credential Response or a Batch Credential Response for the Wallet to be able to use this Endpoint. Support for this endpoint is OPTIONAL.
```

Comment:
```suggestion
This endpoint is used by the Wallet to notify the Credential Issuer of certain events for issued credentials. These events enable the Credential Issuer to take subsequent actions after issuance. The Credential Issuer needs to return one or more notification ids in a Credential Response or a Batch Credential Response for the Wallet to be able to use this Endpoint. Every notification id is related to exactly one issued Credential. Support for this endpoint is OPTIONAL.
```

Reply:
Doesn't this belong in the definition of "notification_id" itself?